infra/resources/_modules/key_vaults/io_com_key_vault.tf

```
@@ -0,0 +1,28 @@
+resource "azurerm_key_vault" "io_com_kv" {
```

```suggestion
resource "azurerm_key_vault" "com" {
```

(it will be referenced as `azurerm_key_vault.com`, it is already explicit, no need to use `io_` and `_kv`)

---

[caeb800](https://github.com/pagopa/io-messages/pull/260/commits/caeb800c25596dec702966dc2e43ab85df5ac94e)